draft-ietf-jsonpath-base.md

```diff
@@ -1113,19 +1106,17 @@ The following table lists filter expression operators in order of precedence fro
 
 The `filter-selector` works with arrays and objects exclusively. Its result might be a list of *zero*, *one*, *multiple* or *all* of their element or member values then. Applied to other value types, it will select nothing.
 
-**FIXME**: The zero number/empty string exceptions are no longer true.  Booleans work the same everywhere.
-
 Negation operator `neg-op` allows to test *falsiness* of values.
```

Comment:
So what about:

Negation operator `neg-op` allows to test for implicit boolean conversion of values resulting in `false`.

Reply:
> Negation operator `neg-op` allows to test *falsiness* of values.

The English is rather stilted, so please can we tidy it up as part of this PR?

I'd suggest something like:
```suggestion
Negation operator `neg-op` of a non-boolean value implicitly converts the value to boolean and then negates the result.
```